spec/features/books/index_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe 'when I visit /books' do
+  context 'as a visitor' do
+    it 'displays all book titles in the database' do
+      stephen_king = Author.create(name: "Stephen King")
+      book_1 = Book.create(title: "IT", page_count: 1168, year: 1986)
+      book_2 = Book.create(title: "The Shining", page_count: 688, year: 1977)
+      BookAuthor.create(author: stephen_king, book: book_1)
+      BookAuthor.create(author: stephen_king, book: book_2)
+
+      visit '/books'
+
+      within "#book-#{book_1.id}" do
```

Comment:
We should also check that the second book is actually being rendered on the page